src/logic/algo/runner.ts

```diff
@@ -0,0 +1,12 @@
+import { TreeBuilder, TreeBuildParams } from "../treeBuilder/treeBuilder";
+import { TreeNode } from "../treeBuilder/treeNode";
+import { MonteCarloTreeSearch, MonteCarloTreeSearchHyperParams } from "./solver";
+
+export type RunParams = TreeBuildParams & MonteCarloTreeSearchHyperParams;
+export class Runner {
```

Comment:
Założenie jest takie, że to odpowiada częściowo odpowiada temu pliku.

https://github.com/wlongxiang/mcts/blob/main/find_max_leaf_binary_tree.py

Z taką różnicą, że pętla 
https://github.com/wlongxiang/mcts/blob/a2ae10a919cf08547968a2895676b71e1ac97547/find_max_leaf_binary_tree.py#L15-L24

powinna moim zdaniem znajdować się nie tutaj, a w klasie która opisuje sam algorytm, ponieważ to są dość powiązane ze sobą rzeczy.

Reply:
Rozumiem. Wtedy solver.run(tree, runParams) zwraca po prostu znalezioną najlepszą (zdaniem algorytmu) wartość?

Reply:
Tak